### ISSUE TITLE
[Suggestion]: Update Applied Energistics 2

### ISSUE THREAD

**Issue by MatusBoa** [FTBTeam/FTB-Modpack-Issues#1501]
### Modpack

FTB Plexiglass Mountain

### Is your suggestion related to a problem? Please describe.

Using latest modpack version 1.2.0. FTB App says there is AE2 with version 11.1.7. Latest version of AE2 is currently 11.5.0 which brings a lot of useful features.

Would it be possible to update?

**Comment by Raidobw2** [comment]
Yes, it's on my todo list. I'll be more available on Wednesday this week, which is where I planned to start more updates for Plexiglass Mountain. I have the habit of making big updates which takes a lot of time, but this time I'll switch my dev habits and push multiple smaller updates as a test as well. I'll add updating AE2 to the first iteration of updates I'll do, thanks for the reminder.

I can't give any ETAs, but as soon as we hit Wednesday that's on my top prio, so it'll be soon after that.

You can leave this opened, I'll close the issue once it has been updated.

**Comment by Raidobw2** [comment]
Fixed in-dev, note that updating AE2 breaks REI compat for now (the "+" in terminals to put the recipe).

On 1.2.1 you'll have the updated AE2 and updated REI

I'll be pushing another update as soon as REI compat with AE2 is working again
